generated/nifgen/enums.py

```diff
@@ -348,24 +326,22 @@ class OutputMode(Enum):
 class P2PAddressType(Enum):
     PHYSICAL = 0
     '''
-    Specifies a physical memory address.
+    Physical
     '''
     VIRTUAL = 1
     '''
-    Specifies a virtual memory address.
+    Physical
```

Comment:
Well this looks wrong. Regardless, it's a P2P attribute so we are removing it.

Reply:
Working on a different PR that will not codegen unreferenced enums.